### PR TITLE
Fixing small bug with finding nearest gridcell for LOCA

### PR DESCRIPTION
**Pull request by claalmve**
# Description of PR

**Summary of changes and related issue**
Fixing small issue in `agnostic.py` to address getting the closest gridcell with LOCA data.

**Relevant motivation and context**
Tool 2 of agnostic tools for finding a specific gridcell does not work with LOCA data.

**Dependencies required for this change?**

**Fixes # (issue), delete if not necessary**

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

**Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.**

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by elehmer — APPROVED** [review]
This works in the current agnostic nb branch. Looks good, just have the one comment.

File: climakitae/explore/agnostic.py [hunk 1]
```diff
@@ -385,9 +385,21 @@ def _compute_results(selections, metric, years, months):
 
     # Retrieving closest grid-cell's data for lat/lon area subsetting
     if selections.area_subset == "lat/lon":
-        data = get_closest_gridcell(
-            data, np.mean(selections.latitude), np.mean(selections.longitude)
-        )
+
+        if (
```
[the remark this answers is not in the document]
Comment (claalmve) [reply climakitae/explore/agnostic.py:389]: Good point, that is cleaner. I'll change it to that.

**Review by Tianchi-Liu — APPROVED** [review]
Merging main should help with the failing check. Otherwise looking good.

**Comment by elehmer** [comment]
> Merging main should help with the failing check. Otherwise looking good.

No need, re-ran the test and good to go. The runner just failed for some unknown reason.

**Comment by claalmve** [comment]
Thank you both!

**Pull request merged**